platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,511 @@
+* The organization has established metrics and goals to observe and target to indicate success.
+
+##### Examples Scenarios:
+
+* The organization has consistently tracked build times and lead time. However, now they realize that while easy to collect, these alone do not give a complete picture of software delivery. With this in mind, the team implements measurement for service reliability and stability.
+
+#### Level 4, Optimizing - Quantitative and qualitative
+
+Feedback and measurements are deeply integrated into the organization's culture. The entire organization, from top-level executives to engineers organization-wide, recognizes the value of data collection and feedback on product evolution. There is a democratization of data, where various stakeholders, including platform users and business leaders, are actively involved in identifying hypotheses for platform improvements, providing feedback during the design process, and then measuring the impact post delivery. All of these measurements are considered when planning platform initiatives.
+
+Not only are standard frameworks leveraged, but there is an understanding that measuring from multiple angles creates a more holistic picture. There is an investment in understanding how qualitative measures change as quantitative ones are improved. There is a focus on identifying leading measures which can allow anticipation of features that would support user needs, alleviate their challenges, and stay ahead of industry trends and business requirements.
+
+##### Characteristics:
+
+* Platform teams continuously seek ways to improve the metrics they watch and the way they gather data
+* Sensitivity to [Goodhart's Law](https://en.wikipedia.org/wiki/Goodhart%27s_law): "When a measure becomes a target, it ceases to be a good measure."
+* Metrics and telemetry gathered is continuously evaluated for true insight and value
+* Metric data management is well supported, such as standard platform capabilities to manage data lakes and derive insights
+* Cross-departmental collaboration is encouraged to avoid data silos and enable effective feedback cycles
+
+##### Examples Scenarios:
+
+* Over time the organization has collected data indicating a rise in build time of over 15%. This triggers negative developer experiences and once triggered, even if the build time is reduced below the original time, developers stay frustrated for longer. This insight drives the build team to set and adhere to a Service Level Objective (SLO), which enables early identification and improvement before instigating the negative cycle with their users.
+
+{{< /tab >}}
+{{< /tabs >}}
+
+</div>
+---
+
+## In summary
+
+Platforms serve as the foundational framework for agile product development. They enable consistency and efficiency in software development and operations. This maturity model can serve as the way to map your platform journey.
```

Comment:
Perhaps we could mention "platform engineering" in this paragraph too to parallel the dual expressions in the introduction. Also perhaps we could allude to our original definition of platforms as "a consistent aggregation of capabilities." Something like the following:

"Platforms and the platform engineers that maintain them serve as a foundation for agile digital product development. They provide a consistent collection of capabilities that enable efficiency in software development and delivery. This maturity model provides a map for your platform engineering journey."